src/overlays/actors/ovl_En_Heishi3/z_en_heishi3.c

```diff
@@ -203,7 +203,7 @@ void func_80A55D00(EnHeishi3* this, GlobalContext* globalCtx) {
     if ((Message_GetState(&globalCtx->msgCtx) == TEXT_STATE_EVENT) && Message_ShouldAdvance(globalCtx) &&
         (this->respawnFlag == 0)) {
         SET_EVENTCHKINF(EVENTCHKINF_4E);
-        globalCtx->nextEntranceIndex = 0x47E; // Hyrule Castle from Guard Capture (outside)
+        globalCtx->nextEntranceIndex = ENTR_SPOT15_4_0;
```

Comment:
I intended to remove all of them, mainly because its not a pattern that we do consistently at all

It does hurt readability short term because the scene names are bad, but that will be fixed eventually. If we had the entrance comments everywhere it might make more sense, but it seems weird to only have them in a few places